Dshop/apps/products_catalogue/filters.py

```diff
@@ -1,11 +1,41 @@
+from django.forms import TextInput
 from django_filters import rest_framework as filters
-from .models import Product
+from .models import Product, Category
 
 
 class ProductFilter(filters.FilterSet):
-    name = filters.CharFilter(field_name='name', lookup_expr='icontains')
-    is_active = filters.BooleanFilter(field_name='is_active')
+    name = filters.CharFilter(field_name='name', lookup_expr='icontains',
```

Comment:
Please add tests.  You can test by checking page context, like here: https://stackoverflow.com/a/50225148
If you don't know how, let me know, I'll guide you.